bin/consequence_mapping/consequence_mapping.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+
+"""
+Pipeline for mapping variants to the genes they affect and their functional consequences, using Ensembl VEP API.
+For documentation, refer to /docs/consequence-mapping.md
+"""
+
+import itertools
+import json
+import logging
+import os
+import requests
+import sys
+
+from retry import retry
+
+logging.basicConfig()
+logger = logging.getLogger('consequence_mapping')
+logger.setLevel(logging.INFO)
+
+
+def deduplicate_list(lst):
+    """Removes duplicates from a list containing arbitrary (possibly unhashable) values."""
+    return [element for element, _ in itertools.groupby(sorted(lst))]
+
+
+def colon_to_vep(colon_id):
```

Comment:
Feel free to expand the function name to **colon_based_id_to_vep_id**. Doesn't cost us anything :D

Reply:
Done, also the same for `vep_to_colon` function